client/apps/plugin-status/view.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLegend from 'components/forms/form-legend';
+import GlobalNotices from 'components/global-notices';
+import HealthView from './health';
+import LogView from './log';
+import notices from 'notices';
+import ServicesStatusView from './services';
+import SettingsGroupCard from 'components/settings-group-card';
+import Toggle from 'components/toggle';
+import {
+	toggleLogging,
+	toggleDebug,
+	save,
+} from './state/actions';
+
+const StatusView = ( { actions, loggingEnabled, debugEnabled, translate } ) => {
+	return (
+		<div>
+			<GlobalNotices id="notices" notices={ notices.list } />
+			<HealthView />
+			<ServicesStatusView />
+			<SettingsGroupCard heading={ translate( 'Debug' ) }>
+				<Toggle
+					id="wcs-toggle-logging"
+					checked={ loggingEnabled }
+					title={ translate( 'Logging' ) }
```

Comment:
The `Debugging` toggle should be displayed before the `Logging` toggle, since after the toggles you have the `Log` textareas.